config/keys.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  apiKey: 'AIzaSyAWtz7gpnbuLlsHI2benyDaUfiKlB-1aqo'
```

Comment:
use dotenv instead with a local .env file; don't commit secrets to git.

 (remove apiKey and rebase to remove from history)

https://www.npmjs.com/package/dotenv
https://12factor.net/config